virttest/utils_test/libvirt.py

```diff
@@ -3932,3 +3933,27 @@ def check_dumpxml(vm, content, err_ignore=False):
                 raise exceptions.TestFail("Expected '%s' was not found in "
                                           "%s's xml" % (content, vm.name))
     return True
+
+
+def checked_setup_or_cleanup_gluster(is_setup, vol_name, brick_path="", pool_name="",
```

Comment:
I am not native, but I can't figure out the meaning of this function name. I'd understand and favor `setup_or_cleanup_gluster` as it does the setup/cleanup (be it from cfg or actual setup) or `check_setup_or_cleanup_gluster` which kind-of loses the setup bit but would be probably understandable. Anyway I am not native so it's possible that this makes more sense so simple answer yes, it makes sense ideally followed by a full sentence I can relate to in the future would be appreciated.

Reply:
what I mean is I'd call them `setup_or_cleanup_gluster` and the original function `_setup_or_cleanup_gluster` or `do_setup_or_cleanup_gluster` to emphasize it's an action. But again, I'm not native so saying no is fine.

Reply:
OK, I meant "checked_function()" to mean "execute function after doing a check", but just using setup_or_cleanup_gluster makes sense